src/stream/dyad_stream_api.hpp

```diff
@@ -187,16 +187,16 @@ bool basic_ifstream_dyad<_CharT, _Traits>::is_open ()
 #else  //-----------------------------------------------------------------------
 template <typename _CharT, typename _Traits>
 basic_ifstream_dyad<_CharT, _Traits>::basic_ifstream_dyad (
-    const dyad_stream_core& ctx)
+    const dyad_stream_core& core)
 {
-    m_ctx.init ();
+    m_core.init ();
```

Comment:
`m_core.init ();` should be removed. In its place, you should have `: m_core (core)` before the opening curly brace.

Reply:
Good catch.